src/screens/SelectionScreen/SelectionScreen.jsx

```diff
@@ -23,6 +23,11 @@ function SelectionScreen(props) {
   function handleSubmit() {
```

Comment:
could we rename this function? or could we move it into one function with a parameter?

Reply:
One fn with param.